mkdir/src/main.rs

```diff
@@ -0,0 +1,49 @@
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+
+use clap::{load_yaml, App};
+
+
+fn log<S: Into<String>>(msg: S) {
+    println!("mkdir: {}", msg.into());
+}
```

Comment:
I didn't quite like this way, since you use for both error and verbose. Verbose should print to `stdout` and error msgs should be printed on `stderr`, this way you only print to `out`

Reply:
I add a log_err that prints to stderr